build_tools/cirrus/update_tracking_issue.sh

```diff
@@ -0,0 +1,20 @@
+# Update tracking issue if Cirrus fails nightly job
+
+if [[ "$CIRRUS_CRON" != "nightly" ]]; then
+    exit 0
+fi
+
+TEST_PASSED="$1"
```

Comment:
Is it possible to document values `TEST_PASSED` can take?

E.g. verbosely:
```suggestion
# TEST_PASSED is either "true" or "false"
TEST_PASSED="$1"
```

What do you think?

Reply:
Yup that makes sense.